Batteries/Data/BitVec/Lemmas.lean

```diff
@@ -0,0 +1,78 @@
+/-
+Copyright (c) 2024 François G. Dorais. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: François G. Dorais
+-/
+
+import Batteries.Data.BitVec.Basic
+
+namespace BitVec
+
+theorem getElem_shiftConcat (v : BitVec n) (b : Bool) (i) (h : i < n) :
```

Comment:
Such basic lemmas about `shiftConcat` should live in core if possible. I would suggest to coordinate with @bollu or @tobiasgrosser on this matter.

Reply:
Yes please.

Reply:
Rather than guessing, any specific place these should be added?

Reply:
Done! [lean4#6875](https://github.com/leanprover/lean4/pull/6875)